tests/e2e/support/page/files/version.ts

```diff
@@ -0,0 +1,28 @@
+import { Actor } from '../../types'
+import { filesCta } from '../../cta'
+
+export class VersionPage {
+  private readonly actor: Actor
+  private readonly fileVersionSidebarXPath: string
```

Comment:
I like it, but I think it's the Locator, not the string. I would suggest to do like here: https://playwright.dev/docs/test-pom